deploy/operator.yaml

```
@@ -41,6 +41,13 @@ spec:
       containers:
         - name: noobaa-operator
           image: NOOBAA_OPERATOR_IMAGE
+          args:
+          - operator  
+          - run       
+          - --health-probe-bind-address=:8081
+          ports:
+            - name: healthz
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)